src/components/TimeSeriesCard/TimeSeriesCard.jsx

```diff
@@ -1,438 +1,142 @@
-const StyledTooltipValue = styled.span`
-  font-size: 0.875rem;
-  font-weight: 400;
-  line-height: 1.125rem;
-  letter-spacing: 0.16px;
-  margin-top: 5px;
-  color: #f3f3f3;
-  font-size: 12px;
-  font-family: IBM Plex Sans;
-`;
-
-class TimeSeriesCard extends Component {
-  constructor(props) {
-    super(props);
-    // Create the ref
-    this.c3ChartComponent = React.createRef();
-    this.state = {
-      colors: {},
-    };
-  }
-
-  // componentDidUpdate(prevProps) {
-  //   // Typical usage (don't forget to compare props):
-  //   console.log('size', this.props);
-  //   if (this.props.size !== prevProps.size) {
-  //     console.log('force resize?');
-  //     this.c3ChartComponent.chart.flush();
-  //   }
-  // }
-
-  componentDidMount() {
-    this.setState({
-      colors: this.c3ChartComponent.chart ? this.c3ChartComponent.chart.data.colors() : {},
-    });
-  }
-
-  customTooltip = (d, defaultTitleFormat, defaultValueFormat, color) => {
-    const content = (
-      <Fragment>
-        <StyledTooltipCaret />
-        {d.map((item, index) => {
-          const dateFormat = new Date(item.x);
-          const titleX = dateFormat.toLocaleDateString('default', {
-            year: 'numeric',
-            month: 'short',
-            day: 'numeric',
-            hour: 'numeric',
-            minute: 'numeric',
-            second: 'numeric',
-          });
-          const valueformat = defaultValueFormat(item.value, item.id, item.index);
-          return (
-            <Fragment>
-              {d.length > 1 ? (
-                <Fragment>
-                  {index === 0 ? <StyledTooltipTitle>{titleX}</StyledTooltipTitle> : null}
-                  <StyledTooltipText color={color(item.id)}>
-                    {item.name}: <StyledTooltipValue>{valueformat}</StyledTooltipValue>
-                  </StyledTooltipText>
-                </Fragment>
-              ) : (
-                <Fragment>
-                  <StyledTooltipText>{titleX}</StyledTooltipText>
-                  <StyledTooltipValue>Value: {valueformat}</StyledTooltipValue>
-                </Fragment>
-              )}
-            </Fragment>
-          );
-        })}
-      </Fragment>
-    );
-
-    return ReactDOMServer.renderToString(content);
+const TimeSeriesCard = ({
+  title,
+  content: { series, timeDataSourceId, xLabel, yLabel },
+  size,
+  interval,
+  values,
+  ...others
+}) => {
+  const sameDay =
+    moment(moment.unix(values[0].timestamp / 1000)).isSame(moment(), 'day') &&
+    moment(moment.unix(values[values.length - 1].timestamp / 1000)).isSame(moment(), 'day');
+
+  const sameYear =
+    moment(moment.unix(values[0].timestamp / 1000)).isSame(moment(), 'year') &&
+    moment(moment.unix(values[values.length - 1].timestamp / 1000)).isSame(moment(), 'year');
+
+  const formatInterval = (timestamp, index) => {
+    const m = moment.unix(timestamp / 1000);
+
+    return sameDay && interval === 'hour' && index === 0
+      ? m.format('DD MMM YYYY')
```

Comment:
are we tracking the localization work somewhere?

Reply:
not at the moment